systemd/systemd_test.go

```diff
@@ -560,6 +566,38 @@ WantedBy=multi-user.target
 	})
 }
 
+func (s *SystemdTestSuite) TestWriteSELinuxMountUnit(c *C) {
+	restore := release.MockSELinuxIsEnabled(func() (bool, error) { return true, nil })
+	defer restore()
+	restore = squashfs.MockUseFuse(false)
+	defer restore()
+
+	mockSnapPath := filepath.Join(c.MkDir(), "/var/lib/snappy/snaps/foo_1.0.snap")
+	err := os.MkdirAll(filepath.Dir(mockSnapPath), 0755)
+	c.Assert(err, IsNil)
+	err = ioutil.WriteFile(mockSnapPath, nil, 0644)
+	c.Assert(err, IsNil)
+
+	mountUnitName, err := New("", nil).AddMountUnitFile("foo", "42", mockSnapPath, "/snap/snapname/123", "squashfs")
+	c.Assert(err, IsNil)
+	defer os.Remove(mountUnitName)
+
+	c.Assert(filepath.Join(dirs.SnapServicesDir, mountUnitName), testutil.FileEquals, fmt.Sprintf(`
+[Unit]
+Description=Mount unit for foo, revision 42
+Before=snapd.service
+
+[Mount]
+What=%s
+Where=/snap/snapname/123
+Type=squashfs
+Options=nodev,ro,x-gdu.hide,context=%s
```

Comment:
Is it ok to pass empty `...context=` on darwin (no unexpected errors from mount?)?

Reply:
I have no clue. We do not reach this code on darwin FWIW.

Reply:
We only pass "context=" if context is != "" (AFAICT from the code above).